src/tribler-core/tribler_core/modules/metadata_store/community/remote_query_community.py

```diff
@@ -0,0 +1,87 @@
+import json
+from binascii import unhexlify
+from random import sample
+
+from ipv8.community import Community
+from ipv8.lazy_community import lazy_wrapper
+from ipv8.messaging.lazy_payload import VariablePayload
+from ipv8.peer import Peer
+from ipv8.requestcache import RequestCache
+
+from tribler_core.modules.metadata_store.orm_bindings.channel_metadata import entries_to_chunk
+
+minimal_blob_size = 200
+maximum_payload_size = 1300
+max_entries = maximum_payload_size // minimal_blob_size
+max_query_peers = 5
+
+
+class RemoteSelectPayload(VariablePayload):
+    format_list = ['I', 'varlenH']
+    names = ['id', 'json']
+
+
+class SelectResponsePayload(VariablePayload):
+    format_list = ['I', 'raw']
+    names = ['id', 'raw_blob']
+
+
+class RemoteQueryCommunity(Community):
+    """
+    Community for general purpose SELECT-like queries into remote Channels database
+    """
+
+    master_peer = Peer(
+        unhexlify(
+            "4c69624e61434c504b3a667b8dee4645475512c0780990cfaca234ad19c5dabcb065751776"
+            "b75a4b4210c06e2eb4d8bbf4a775ed735eb16bbc3e44193479ad7426d7cd1067807f95b696"
+        )
+    )
+
+    SELECT_REQUEST = 1
```

Comment:
What I do lately is add a `msg_id` field to the variable payloads instead of using community class fields.

Something like:

```python
@vp_compile
class SelectResponse(VariablePayload):
    format_list = ['I', 'raw']
    names = ['id', 'raw_blob']
    msg_id = 2

[...]

self.ezr_pack(SelectResponse.msg_id, SelectResponse(request.id, data)
```

It's not shorter, but it keeps everything in the same place (and if I add a shortcut for it in the future you're immediately compatible 😃 ).